Sources/Starknet/Providers/StarknetProvider/StarknetProvider.swift

```diff
@@ -7,6 +7,8 @@ public enum StarknetProviderError: Error {
     case emptyBatchRequestError
 }
 
+@available(macOS 13.0.0, *)
+@available(iOS 16.0.0, *)
```

Comment:
note: Using parametrized protocol requires adding `@available` attributes, respectively for these version.

Reply:
Good catch, changed.